localization/vi/aggregator-microservices/README.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Aggregator Microservices (Microservices Tổng Hợp)
```

Comment:
I suggest using `Microservices Tổng Hợp`

Reply:
Okay, I'll use `Microservices Tổng Hợp` as a main title (instead of using it as a title note)